2019-08-23 18:25:56,897 [pyphi.conf] INFO MainProcess: PyPhi v1.2.0
2019-08-23 18:25:56,897 [pyphi.conf] INFO MainProcess: Using default configuration (no configuration file provided)
2019-08-23 18:25:56,898 [pyphi.conf] INFO MainProcess: Current PyPhi configuration:
 { 'ASSUME_CUTS_CANNOT_CREATE_NEW_CONCEPTS': False,
  'CACHE_POTENTIAL_PURVIEWS': True,
  'CACHE_REPERTOIRES': True,
  'CACHE_SIAS': False,
  'CACHING_BACKEND': 'fs',
  'CLEAR_SUBSYSTEM_CACHES_AFTER_COMPUTING_SIA': False,
  'CUT_ONE_APPROXIMATION': False,
  'FS_CACHE_DIRECTORY': '__pyphi_cache__',
  'FS_CACHE_VERBOSITY': 0,
  'LOG_FILE': 'pyphi.log',
  'LOG_FILE_LEVEL': 'INFO',
  'LOG_STDOUT_LEVEL': 'WARNING',
  'MAXIMUM_CACHE_MEMORY_PERCENTAGE': 50,
  'MEASURE': 'EMD',
  'MONGODB_CONFIG': { 'collection_name': 'cache',
                      'database_name': 'pyphi',
                      'host': 'localhost',
                      'port': 27017},
  'NUMBER_OF_CORES': -1,
  'PARALLEL_COMPLEX_EVALUATION': False,
  'PARALLEL_CONCEPT_EVALUATION': False,
  'PARALLEL_CUT_EVALUATION': True,
  'PARTITION_TYPE': 'BI',
  'PICK_SMALLEST_PURVIEW': False,
  'PRECISION': 6,
  'PRINT_FRACTIONS': True,
  'PROGRESS_BARS': True,
  'REDIS_CACHE': False,
  'REDIS_CONFIG': {'db': 0, 'host': 'localhost', 'port': 6379, 'test_db': 1},
  'REPR_VERBOSITY': 2,
  'SINGLE_MICRO_NODES_WITH_SELFLOOPS_HAVE_PHI': False,
  'SYSTEM_CUTS': '3.0_STYLE',
  'USE_SMALL_PHI_DIFFERENCE_FOR_CES_DISTANCE': False,
  'VALIDATE_CONDITIONAL_INDEPENDENCE': True,
  'VALIDATE_SUBSYSTEM_STATES': True,
  'WELCOME_OFF': False}
2019-08-23 18:27:10,334 [pyphi.conf] INFO MainProcess: PyPhi v1.2.0
2019-08-23 18:27:10,334 [pyphi.conf] INFO MainProcess: Using default configuration (no configuration file provided)
2019-08-23 18:27:10,335 [pyphi.conf] INFO MainProcess: Current PyPhi configuration:
 { 'ASSUME_CUTS_CANNOT_CREATE_NEW_CONCEPTS': False,
  'CACHE_POTENTIAL_PURVIEWS': True,
  'CACHE_REPERTOIRES': True,
  'CACHE_SIAS': False,
  'CACHING_BACKEND': 'fs',
  'CLEAR_SUBSYSTEM_CACHES_AFTER_COMPUTING_SIA': False,
  'CUT_ONE_APPROXIMATION': False,
  'FS_CACHE_DIRECTORY': '__pyphi_cache__',
  'FS_CACHE_VERBOSITY': 0,
  'LOG_FILE': 'pyphi.log',
  'LOG_FILE_LEVEL': 'INFO',
  'LOG_STDOUT_LEVEL': 'WARNING',
  'MAXIMUM_CACHE_MEMORY_PERCENTAGE': 50,
  'MEASURE': 'EMD',
  'MONGODB_CONFIG': { 'collection_name': 'cache',
                      'database_name': 'pyphi',
                      'host': 'localhost',
                      'port': 27017},
  'NUMBER_OF_CORES': -1,
  'PARALLEL_COMPLEX_EVALUATION': False,
  'PARALLEL_CONCEPT_EVALUATION': False,
  'PARALLEL_CUT_EVALUATION': True,
  'PARTITION_TYPE': 'BI',
  'PICK_SMALLEST_PURVIEW': False,
  'PRECISION': 6,
  'PRINT_FRACTIONS': True,
  'PROGRESS_BARS': True,
  'REDIS_CACHE': False,
  'REDIS_CONFIG': {'db': 0, 'host': 'localhost', 'port': 6379, 'test_db': 1},
  'REPR_VERBOSITY': 2,
  'SINGLE_MICRO_NODES_WITH_SELFLOOPS_HAVE_PHI': False,
  'SYSTEM_CUTS': '3.0_STYLE',
  'USE_SMALL_PHI_DIFFERENCE_FOR_CES_DISTANCE': False,
  'VALIDATE_CONDITIONAL_INDEPENDENCE': True,
  'VALIDATE_SUBSYSTEM_STATES': True,
  'WELCOME_OFF': False}
2019-08-26 11:01:46,349 [pyphi.conf] INFO MainProcess: PyPhi v1.2.0
2019-08-26 11:01:46,349 [pyphi.conf] INFO MainProcess: Using default configuration (no configuration file provided)
2019-08-26 11:01:46,350 [pyphi.conf] INFO MainProcess: Current PyPhi configuration:
 { 'ASSUME_CUTS_CANNOT_CREATE_NEW_CONCEPTS': False,
  'CACHE_POTENTIAL_PURVIEWS': True,
  'CACHE_REPERTOIRES': True,
  'CACHE_SIAS': False,
  'CACHING_BACKEND': 'fs',
  'CLEAR_SUBSYSTEM_CACHES_AFTER_COMPUTING_SIA': False,
  'CUT_ONE_APPROXIMATION': False,
  'FS_CACHE_DIRECTORY': '__pyphi_cache__',
  'FS_CACHE_VERBOSITY': 0,
  'LOG_FILE': 'pyphi.log',
  'LOG_FILE_LEVEL': 'INFO',
  'LOG_STDOUT_LEVEL': 'WARNING',
  'MAXIMUM_CACHE_MEMORY_PERCENTAGE': 50,
  'MEASURE': 'EMD',
  'MONGODB_CONFIG': { 'collection_name': 'cache',
                      'database_name': 'pyphi',
                      'host': 'localhost',
                      'port': 27017},
  'NUMBER_OF_CORES': -1,
  'PARALLEL_COMPLEX_EVALUATION': False,
  'PARALLEL_CONCEPT_EVALUATION': False,
  'PARALLEL_CUT_EVALUATION': True,
  'PARTITION_TYPE': 'BI',
  'PICK_SMALLEST_PURVIEW': False,
  'PRECISION': 6,
  'PRINT_FRACTIONS': True,
  'PROGRESS_BARS': True,
  'REDIS_CACHE': False,
  'REDIS_CONFIG': {'db': 0, 'host': 'localhost', 'port': 6379, 'test_db': 1},
  'REPR_VERBOSITY': 2,
  'SINGLE_MICRO_NODES_WITH_SELFLOOPS_HAVE_PHI': False,
  'SYSTEM_CUTS': '3.0_STYLE',
  'USE_SMALL_PHI_DIFFERENCE_FOR_CES_DISTANCE': False,
  'VALIDATE_CONDITIONAL_INDEPENDENCE': True,
  'VALIDATE_SUBSYSTEM_STATES': True,
  'WELCOME_OFF': False}
2019-08-26 11:23:53,676 [pyphi.compute.subsystem] INFO MainProcess: Calculating big-phi data for Subsystem(S1, S2, M1, M2, H1, H2, H3, H4)...
2019-08-26 11:23:53,678 [pyphi.compute.subsystem] INFO MainProcess: Subsystem(S1, S2, M1, M2, H1, H2, H3, H4) is not strongly connected; returning null SIA immediately.
2019-08-26 11:29:44,789 [pyphi.compute.subsystem] INFO MainProcess: Calculating big-phi data for Subsystem(S1, S2, M1, M2, H1, H2, H3, H4)...
2019-08-26 11:29:44,789 [pyphi.compute.subsystem] INFO MainProcess: Subsystem(S1, S2, M1, M2, H1, H2, H3, H4) is not strongly connected; returning null SIA immediately.
2019-08-26 11:29:46,897 [pyphi.compute.subsystem] INFO MainProcess: Calculating big-phi data for Subsystem(H4)...
2019-08-26 11:29:46,898 [pyphi.compute.subsystem] INFO MainProcess: Single micro nodes Subsystem(H4) with selfloops cannot have phi; returning null SIA immediately.
